### PR TITLE
Copper PHY Attributes for MDIX, Module type, Autoneg mode, Latency mode, Dual Media mode

### PR DESCRIPTION
**Pull request by SidharajU**
The original PR for this change was https://github.com/opencomputeproject/SAI/pull/1217. I have created a new PR based on the feedback received from SAI community and will abandon the original PR.

### CODE REVIEW

File: inc/saiport.h [hunk 1]
```diff
@@ -1713,6 +1796,74 @@ typedef enum _sai_port_attr_t
...
+     */
+    SAI_PORT_ATTR_EEE_LATENCY_MODE,
+
+    /**
+     * @brief Configure Fiber module type
+     *
+     * @type sai_port_module_type_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MODULE_TYPE_1000BASE_X
+     * @condition SAI_PORT_ATTR_SPEED == 1000 or SAI_PORT_ATTR_SPEED == 100 or SAI_PORT_ATTR_MEDIA_TYPE == SAI_PORT_ATTR_MEDIA_TYPE_FIBER
```
Comment (kcudnik) [inline inc/saiport.h:1854]: value for SAI_PORT_ATTR_MEDIA_TYPE  must be one of SAI_PORT_MEDIA_TYPE_*

Comment (kcudnik) [reply inc/saiport.h:1854]: also condition attribute must be mandatory on create, or you can change this to validonly

Comment (SidharajU) [reply inc/saiport.h:1854]: So there is no way to have conditions with multiple attributes. As discussed last week, we would like to allow this attribute when media type is set to fiber and speed is either 1000 or 100. How do I do that?

**Review by kcudnik — APPROVED** [review]

**Comment by lguohan** [comment]
@jleveque , can you check this pr?

**Review by jleveque — CHANGES_REQUESTED** [review]
Proposal should be written in Markdown language. Also please fix conflicts.

**Comment by SidharajU** [comment]
There were some issues with the merge so I am creating a new PR. I don't know how to recover this PR now.

**Comment by SidharajU** [comment]
This PR has merge issues so created another PR
https://github.com/opencomputeproject/SAI/pull/1248

**Pull request closed without merging**